modules/vaos/app/services/vaos/systems_service.rb

```diff
@@ -7,7 +7,10 @@ class SystemsService < VAOS::BaseService
     def get_systems
       with_monitoring do
         response = perform(:get, '/mvi/v1/patients/session/identifiers.json', nil, headers)
-        response.body.map { |system| OpenStruct.new(system) }
+        response
+          .body
+          .select { |system| system[:assigning_authority].include?('dfn-') || system[:assigning_code].include?('CRNR') }
```

Comment:
We would need to allow Cerner ids through in the future, so I made an attempt at that now. Could remove it if we don't want to go there yet.